apps/teams-test-app/e2e-test-data/copilot.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "copilot",
+  "platforms": "*",
+  "version": ">=2.22.0",
+  "testCases": [
+    {
+      "title": "copilot hasM365Chat license API Call - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_CheckIfUserOfTheM365ChatAppHasLicense",
+      "expectedTestAppValue": "User does have a license for the M365 Chat app"
+    }
```

Comment:
We should have a test that returns false as well, right?

Reply:
We usually don't have failure E2E tests. Also, how can we have both failure and success run at the same time?